portal/templates/gil/what-is-prostate-cancer.html

```diff
@@ -1,144 +1,144 @@
 {% extends "gil/base.html" %}
-{% block title %}Truenth Prostate Cancer Information{% endblock %}
+{% block title %}{{_("TrueNTH Prostate Cancer Information")}}{% endblock %}
 {% block head %}{{super()}}{% endblock %}
 {% block main %}
     <main class="main">
       {{ super() }}
       <div class="page-intro page-intro--dark page-intro--top-img page-intro--prostate-facts" style="background-image: url({{url_for('static', filename='img/PCF2-Background.jpg') }})">
         <div class="page-intro__content">
-          <h1 class="page-intro__main">What is <span class="page-intro__main--accent">Prostate Cancer?</span></h1>
-          <p class="page-intro__sub">Cancer is a disease in which cells in the body grow out of control. Prostate Cancer is when cancer starts in the prostate. Many men with prostate cancer die of other causes without ever having any symptoms from the cancer.</p>
-          <div class="button-callout button-callout--has-text"><strong class="js-scroll-down" data-target=".page-intro">Learn More</strong>
+          <h1 class="page-intro__main"><span>{{_("What is")}}<span> <span class="page-intro__main--accent">{{_("Prostate Cancer?")}}</span></h1>
```

Comment:
This should probably be in a trans block (including HTML as necessary) since it's a complete question/sentence